src/server/api.js

```diff
@@ -159,7 +159,7 @@ export const createApiServer = ({ db, games }) => {
     if (!gameMetadata.players[playerID]) {
       ctx.throw(404, 'Player ' + playerID + ' not found');
     }
-    if (gameMetadata.players[playerID].name) {
```

Comment:
Actually, maybe a cleaner way would be to use:
```
gameMetadata.players[playerID].name = playerName;
gameMetadata.players[playerID].occupied = true;
```

down below at line 166

and then you can just check:

```
if (gameMetadata.players[playerID].occupied) {
  ctx.throw(409, 'Player ' + playerID + ' already occupied');
}
```

here.

That would give us flexibility to also have slots without a name.

Reply:
Breaking changes are OK for an alpha release.  I'll just make sure to document this carefully in the changelog.